src/react/components/common/keyboardManager/keyboardRegistrationManager.test.ts

```diff
@@ -31,30 +32,55 @@ describe("Keyboard Registration Manager", () => {
         expect(handlers2[0]).toBe(handler2);
     });
 
+    it("can register handlers for same key code and different key event types", () => {
+        const keyCode = "Ctrl+H";
```

Comment:
Is H just random letter you're using to test this or is there a reason for it?

Reply:
Yep, just random :)